src/test/test_confparse.c

```diff
@@ -46,15 +48,17 @@ typedef struct test_struct_t {
 
 static test_struct_t test_struct_t_dummy;
 
-#define VAR(name,conftype,member,initvalue)                             \
-  { name, CONFIG_TYPE_##conftype, offsetof(test_struct_t, member),      \
+#define VAR(varname,conftype,member,initvalue)                          \
```

Comment:
These are also duplicates, which should get removed at some future time.
(Unless they are necessary for low-level tests. If they are, they should get a comment saying so.)

Reply:
This also gets fixed when we combine everything into serc/lib/confmacros.h in c553750 as a part of #30935.